portfolio/src/main/webapp/script.js

```diff
@@ -63,15 +63,53 @@ function showComments() {
         document.getElementById("thecomments").innerHTML += "<p>" + 
         jsoncomments[jsoncomments.length - 1] + "</p";
     })
+    loadComments();
 }
 
 /**
  * Accesses the comments from the datastore and displays them.
  */
 function loadComments() {
     fetch("/data").then(dataHashMap => dataHashMap.json()).then(dataHashMapJson => {
-        for (comment of dataHashMapJson.Comments) {
-            document.getElementById("thecomments").innerHTML += "<p>" + comment + "</p";
+        if (document.getElementById("CommentNumber").options[0].selected === true) { //5
```

Comment:
Ah. I didn't know that. I'll make the change.

Reply:
Sorry there's a typo in my previous comment:

// I mean to use "commentNumber" in forloop instead of "5".
// Main purpose was trying to reduce duplication in code.
for (let i = 0; i < commentNumber; i++) {
...
}

Putting it altogether:

const len = dataHashMapJson.Comments.length;
let commentNumber = math.Min(document.getElementById("CommentNumber").options[0].selected ? 5 : 10, len);
// Choose the min of len and selected number so you don't need to do additional if condition in loop.

for (let i = 0; i < commentNumber; i++) {
...
}